ilastik/applets/dataSelection/opDataSelection.py

```diff
@@ -63,14 +62,28 @@ def __init__(self, file_path: str, base_dir: str):
         super().__init__(f"Can't represent {file_path} relative to {base_dir}")
 
 
+class InconsistentAxisMetaException(Exception):
+    def __init__(self, axistags: AxisTags, shape):
+        if len(axistags) > len(shape):
+            problem = "More axes than data dimensions."
+        else:
+            problem = "Some data dimensions have no axis interpretation."
+        super().__init__(
+            f"Unable to load: {problem}. Please check if all files have the same dimensionality.\n"
+            f"Reported axes: {', '.join([tag.key for tag in axistags])}\n"
+            f"Actual data dimensions: {', '.join([str(s) for s in shape])} (pixels/channels/timepoints)"
+        )
+
+
 class UnsuitedAxistagsException(Exception):
     def __init__(self, axistags: AxisTags, shape):
         if len(axistags) > len(shape):
-            problem = "Inconsistent metadata: The dataset reports dimensions for which it contains no data."
+            problem = "No data for some axes."
         else:
-            problem = "Inconsistent metadata: The dataset has more dimensions than it reports."
+            problem = "Some data dimensions have no axis interpretation."
```

Comment:
I know you didn't really change the logic here, but I would expect a single channel time series with all axistags to work in a project that was trained on a single channel xy image...

In dataselection those two datasets would coexist happily.


![image](https://github.com/ilastik/ilastik/assets/24434157/2b720af1-24c6-4bf6-a332-d3b2f7f9dcfe)

Reply:
You mean they should coexist happily but currently don't..? I'd say make an issue for it...

Reply:
k, let's open an issue, it's unintuitive that it can be added as input data, but not in batch.